ports/rp2/boards/W5500_EVB_PICO/readme.md

```diff
@@ -0,0 +1,15 @@
+# Wiznet W5500-EVB-Pico
```

Comment:
if this is named `board.md` then it'll appear automatically at the download page on micropython.org

Reply:
Sorry I don't not get this comment. I just copied the file from the WS5100S_EVB_Pico directory and updated it.